cluster.go

```diff
@@ -17,10 +18,11 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
 )
 
-// Common errors
-var (
-	ErrBootstrap = errors.New("bootstrap unsuccessful")
-)
+// ReadyTimeout specifies the  before giving up
+// during startup (waiting for consensus to be ready)
+// It may need adjustment according to timeouts in the
+// consensus layer.
+var ReadyTimeout = 30 * time.Second
```

Comment:
Would it make sense to have this configurable?  Depending on latency between nodes certain clusters might always want a higher readytimeout.

Reply:
Note it is configurable, in a sense. cluster-service sets it in relation to wait_for_leader_timeout. Exposing it in config complicates the config and allows misconfiguring it, so I left it.

Before it was exacty the same, but now the main component doesn't have access to the consensusCfg.